packages/ethereumjs-config-tslint/tslint.json

```diff
@@ -1,13 +1,17 @@
 {
   "extends": "typestrict",
   "rules": {
-    "no-console": true,
+    "no-console": {
+      "severity": "warning"
+    },
```

Comment:
These seems to be the legacy way of configuring things, see https://palantir.github.io/tslint/usage/configuration/ (search for `severity`), can we go with the new syntax here?

Reply:
Sorry I couldn't find what you're referring to?

Reply:
Sorry, I misread this in the docs. Re-read, your version is actually the new one, will approve.